addOns/pscanrules/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-- View State Scan Rules (Issue 8262)
+- References for Alerts generated by the View State scan rule were updated (Issue 8262).
```

Comment:
Still needs the `### Changed` heading.

Reply:
ok so it's like 
## Unreleased
### Changed 
- References for Alerts generated by the View State scan rule were updated (Issue 8262).

Reply:
It should be added to the existing entry.

Reply:
under changed section ?

Reply:
Yes, around line 15 (for current changes).

Reply:
Sorry I guess I was misleading, I hadn't checked what was already updated in the current CHANGELOG.md.

Reply:
No problem 
thanks for help 🙇‍♂️